### PR TITLE
Added a little help in README.md

### PR DESCRIPTION
**Pull request by jackctj117**
Added contents to README.md, only added a little mostly just mentioning what the example is and making sure the users Makefile has the correct directory name for wolfHSM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,90 @@
 # wolfHSM-examples
 
-## Building the Posix Example
+## Current Examples
 
-<intro on what the posix example is>
+### Posix TCP server and client
```
Comment (bigbrett) [inline README.md:5]: @jackctj117 could you provide a little blurb summarizing what the POSIX TCP server and client is/does? (e.g. spins up a wolfHSM server in the server thread, client in the client thread, then runs the unit tests on the client side against the server)

File: README.md [hunk 2]
```diff
@@ -1,18 +1,90 @@
...
 ### Building wh_server_tcp
+cd into `wolfHSM-examples/posix/tcp/wh_server_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_server_tcp.elf`.
+
+### Building wh_client_tcp
+Building the client will follow the same steps as the building the server. cd into `wolfHSM-examples/posix/tcp/wh_client_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_client_tcp.elf`.
 
-<steps to build wh_server_tcp>
+### Executables
+Once both the server and the client `.elf` files have been made the example can now be ran. In the root directory for wolfHSM-examples run `./posix/tcp/wh_server_tcp/Build/wh_server_tcp.elf`.
+In a sperate instnace run `./posix/tcp/wh_client_tcp/Build/wh_client_tcp.elf` out of the root directory for wolfHSM-examples for the client.
```
Comment (bigbrett) [inline README.md:28]:
```suggestion
In a sperate instance run `./posix/tcp/wh_client_tcp/Build/wh_client_tcp.elf` out of the root directory for wolfHSM-examples for the client.
```

might be a few other Typos too

File: README.md [hunk 3]
```diff
@@ -1,18 +1,90 @@
...
+```
+# Important directories
+BUILD_DIR = ./Build
+WOLFHSM_DIR = $(CURDIR)/../../../../wolfHSM
 
+WOLFSSL_DIR ?= $(CURDIR)/../../../../wolfssl
+```
+After making sure the `WOLFHSM_DIR` matches the directory name downloaded for wolfHSM the next step is building the client and server.
 ### Building wh_server_tcp
+cd into `wolfHSM-examples/posix/tcp/wh_server_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_server_tcp.elf`.
```
Comment (bigbrett) [inline README.md:21]:
```suggestion
`cd` into `wolfHSM-examples/posix/tcp/wh_server_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_server_tcp.elf`.
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,90 @@
...
+
+### Building wh_client_tcp
+Building the client will follow the same steps as the building the server. cd into `wolfHSM-examples/posix/tcp/wh_client_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_client_tcp.elf`.
```
Comment (bigbrett) [inline README.md:24]:
```suggestion
Building the client will follow the same steps as the building the server. `cd` into `wolfHSM-examples/posix/tcp/wh_client_tcp`, next `make`. Once completed an `.elf` executable will be made in `Build` named `wh_client_tcp.elf`.
```

**Review by bigbrett — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jackctj117) [reply README.md:5]: Added a short description and fixed all other minor request.

**Review by bigbrett — APPROVED** [review]
Thanks @jackctj117

**Pull request merged**